src/authorization/authorization.ts

```diff
@@ -767,6 +769,20 @@ export function initialize(
         /* stop adding JWT to requests */
         baseAxiosRequest.interceptors.request.eject(userInterceptor);
       }
+    },
+    refreshJwtToken: async (authType: string) => {
```

Comment:
i think `userId` might be more applicable here as opposed to auth type since it will be a user email or UUID.

Reply:
or maybe just `user`? 🤔